src/Components/ToDoItem/ToDoItem.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import { default as toDoItemStyles} from './ToDoItem.module.scss';
 import bemCssModule from 'bem-css-modules';
+import { ToDo } from '../../Constants/types';

```

Comment:
You have to use absolute path ( src/constants/............. ) instead ( ../../constants/.......... ).
Folder name should start with lower case unless it is a Component folder name.